src/sentry/utils/auth.py

```diff
@@ -367,3 +368,9 @@ def authenticate(self, request: Request, username=None, password=None):
 
     def user_can_authenticate(self, user):
         return True
+
+
+def make_login_link_with_redirect(path, redirect):
+    query_string = urlencode({REDIRECT_FIELD_NAME: redirect})
+    redirect_uri = f"{path}?{(query_string)}"
```

Comment:
```suggestion
    redirect_uri = f"{path}?{query_string}"
```
Don't know if these parens are needed here.